client/src/document/index.scss

```diff
@@ -958,30 +952,37 @@ html a.only-in-en-us:after {
 
 html[lang="de"] a.only-in-en-us:after {
   content: "(engl.)";
+  margin-left: 0.5ch;
```

Comment:
Rather than duplicating, let's move this up to the `tml a.only-in-en-us:after { ... }` rule, and overriding for `zh-CN` and `zh-TW` (e.g. `margin-left: unset`).

Reply:
Done :ok_hand: